src/Writer/XLSX/Options.php

```diff
@@ -63,4 +70,103 @@ public function getMergeCells(): array
     {
         return $this->MERGE_CELLS;
     }
+
+    /**
+     * set Worksheets page margins.
+     *
+     * @param float $top    inches
+     * @param float $right  inches
+     * @param float $bottom inches
+     * @param float $left   inches
+     * @param float $header inches
+     * @param float $footer inches
+     */
+    public function setPageMargins(
+        float $top = 0.75,
+        float $right = 0.7,
+        float $bottom = 0.75,
+        float $left = 0.7,
+        float $header = 0.3,
+        float $footer = 0.3
```

Comment:
Let's use a dedicated object like the [`MergeCell`](https://github.com/openspout/openspout/blob/v4.16.0/src/Writer/XLSX/MergeCell.php) one

Reply:
We can directly require a `PageMargin` object as the argument